code/modules/mob/living/carbon/human/species_types/monkeys.dm

```diff
@@ -35,6 +35,7 @@
 	gib_anim = "gibbed-m"
 
 	payday_modifier = 1.5
+	var/can_gorrilize = TRUE
```

Comment:
you forgot to set this to FALSE for kobolds

note to self: make some sort of dragon-like creature the gorilla equivalent for kobolds